### DISCUSSION TITLE
Impossible to apply to Github Education

### DISCUSSION THREAD

**Discussion by Lev-Shapiro** [community/community#49361] [GitHub Education]
I've submitted already two times that form and in both of them got rejected. As I think, that's probably because of Github's bug in finding schools...

On the page where it requires to find my school on the map - I simply can't find it!!!! I live in Israel and I've already submitted that issue to Github Education support, but the problem is that they ignored me and wrote something like "try applying using this link: {link}"!

I've just wasted my time! 2 hours on that stuff! As a result I got nothing. I have no idea why you simply can't open Google Maps, enter that school name which I wrote from the first page and notice that your system has a bug - not rendering as expected all those schools! 

Why is that so hard? You reject even if that's your problem... If this message won't help I won't try anymore. I have stuff to do other than screening my school's website, lesson schedule and all that stuff.

For the end I'll say that if you would be really trying to help as you say, you wouldn't be writing SAME message everytime you reject someone!

**Comment by samus-aran** [comment]
Hi @Lev-Shapiro 👋 

We're happy you're here, lets see how we can assist. As you can imagine there are many types of academic verification (and schools) and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

The application does ask you to first find a campus local to you based on your location. If you are unable to find one it then asks you to identify that school with additional information e.g. school website, location. This usually happens when the school you are applying for has not been found.

Alternatively you can ask a teacher at your school to apply as a [GitHub School Partner](https://education.github.com/schools) which will help register your school.  

Can you describe in steps what section you are failing to identify your school?

**Reply by Lev-Shapiro** [reply]
Hi @samus-aran,

First of all, wouldn't moving from strange API for finding schools to something like Google Maps API solve the problem?

Secondly, you've wrote that if I couldn't find it - it will ask me to identify my school with additional information. The problem - it doesn't! It does have those fields, but if I don't fill where my school is located on the map then it doesn't allow me to continue to next step, so I simply click randomly one of options near me because what else can I do?! I've wrote to Github Support about that and got very weird message that didn't help me at all... Anyway, when I click on one of those options the location I've wrote for my school automatically changes to what I've clicked.

The even bigger problem is that I do have the option to fill my school website and I do it, but for unknown to me reasons they even don't check it and they already 2 times couldn't understand that it's not my problem, but bug of Github's Education form that I can't normally fill that form with valid information... 

In one of those sentences of rejection message they write "Please provide the name of your school in both languages", which I did, but as I understood, the problem is not with the first page where it requires me to enter school name, but in the second, where they find that because of their bug 1-2 fields are incorrect...

I can't really understand why proving that you are a student is so hard in Github Education... Oh, and if you really want to check I'm student - let's make a video call!

**Reply by Lev-Shapiro** [reply]
@samus-aran could you answer please?

**Reply by ghost** [reply]
Having the exact same problem, have been waiting for support's answer for 2 weeks already.

**Reply by subh63** [reply]
I've applied more than 20 times with two different accounts and variious other school details but nothing seems to work , It's just absolutely frustrating !!!

**Reply by FranzLor** [reply]
i swear they make this impossible just to gatekeep you from getting free benefits
